test/automation/src/positron/positronNotebooks.ts

```diff
@@ -53,6 +53,15 @@ export class PositronNotebooks {
 
 	async executeInFirstCell(code: string) {
 		await this.quickaccess.runCommand(EDIT_CELL_COMMAND);
+
+		if (process.platform === 'darwin') {
+            await this.code.dispatchKeybinding('cmd+A');
+        }
+        else {
+            await this.code.dispatchKeybinding('ctrl+A');
+        }
+		await this.code.dispatchKeybinding('Delete');
+
```

Comment:
Yeah, the formatting doesn't look like that in the code.  Not sure whats going on with github.  Adding a comment about why we are doing this: because in retry cases we want to be sure to clear out anything that was already in the cell.